api/locale/en-us.yml

```diff
@@ -0,0 +1,20 @@
+en-us:
+  views: "%{number} views"
+  seconds-ago:
+    one: "%{count} second ago"
+    many: "%{count} seconds ago"
+  minutes-ago:
+    one: "1 minute ago"
+    many: "%{count} minutes ago"
```

Comment:
It seems like there is a bit of an inconsistency with the `one`s as to whether `%{count}` is used or `1`. Either way is fine, but it should be consistent across all of them. I think doing `1 second ago` instead of `%{count} second ago` would be fine.